### DISCUSSION TITLE
Percentage based on two columns

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#314] [Q&A] [answered]
Greetings

Using csv inline,  how can I find the % difference between two values?

I used the Condition to divide current vs totalvalue ie 0.70 bit need that as 70%

Thanks

**Comment by yesoreyeram** [comment, marked as answer]
Unfortunately, there is no direct manipulation for this. Either you can go with grafana transformations or using uql, you can achieve this. 

```csv
itemName,currentValue,totalValue
mango,30,70
apple,20,100
```

```kusto
parse-csv
| extend "pct_value"=div("currentValue","totalValue")
| extend "pct_value"=mul("pct_value",100)
| project-away "currentValue", "totalValue"
```

Still this is not intuitive way ( ideally, we want to get it in single command though.) But [it](https://grafana-infinity-datasource.herokuapp.com/explore?orgId=1&left=%7B%22datasource%22:%22Infinity%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22type%22:%22uql%22,%22source%22:%22inline%22,%22format%22:%22table%22,%22url%22:%22https:%2F%2Fjsonplaceholder.typicode.com%2Fusers%22,%22url_options%22:%7B%22method%22:%22GET%22,%22data%22:%22%22%7D,%22root_selector%22:%22%22,%22columns%22:%5B%5D,%22filters%22:%5B%5D,%22global_query_id%22:%22%22,%22data%22:%22itemName,currentValue,totalValue%5Cnmango,30,70%5Cnapple,20,100%22,%22uql%22:%22parse-csv%5Cn%7C%20extend%20%5C%22pct_value%5C%22%3Ddiv(%5C%22currentValue%5C%22,%5C%22totalValue%5C%22)%5Cn%7C%20extend%20%5C%22pct_value%5C%22%3Dmul(%5C%22pct_value%5C%22,100)%5Cn%7C%20project-away%20%5C%22currentValue%5C%22,%20%5C%22totalValue%5C%22%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) works for now.

**Comment by yosiasz** [comment]
It works is the key word, thanks much!
